modules/library/cql/ECQL.md

```diff
@@ -37,8 +37,8 @@ is used to describe the grammar.
 
 ```
     <sequence of search conditions> ::=
-                <search condition>
-                   |<sequence of search conditions> <semicolon> <search condition>
+        <search condition>
+        | <sequence of search conditions> <semicolon> <search condition>
```

Comment:
Just making the indentation consistent with the rest of the document. Feel free to revert.